src/renderer/components/+workloads-deployments/deployment-scale-dialog.scss

```diff
@@ -20,11 +20,17 @@
     }
 
     .desired-scale {
-      flex: 1 0;
+      flex: 1.1 0;
     }
 
     .slider-container {
-      flex: 1.3 0;
+      flex: 1 0;
+    }
+
+    .plus-minus-container {
+      margin-left: 15px;
```

Comment:
For most cases we're using predefined variables for paddings and margins. See here `src/renderer/components/vars.scss`.
So current margin could look like `margin-left: $margin * 2`